Automattic/jetpack
# My Jetpack: Make whole Product card clickable

This PR makes the whole Card Product clickable when the product requires it.

Fixes https://github.com/Automattic/jetpack/issues/23484

#### Changes proposed in this Pull Request:

* My Jetpack: Make whole Product card clickable

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to My Jetpack
* Test with some products that require be added
* Confirm it leads to the interstitial page by clicking on the whole card
* Confirm the cards are navigable by keyboard like other regular links.
* Confirm the focus styles

https://user-images.githubusercontent.com/77539/158836543-8f48a7ef-b03a-417a-95bf-535a838d266d.mov

With Storybook

https://user-images.githubusercontent.com/77539/158837193-6e7943cc-aad6-47e5-a634-71bae71818a4.mov
